### ISSUE TITLE
Cannot play AudioClips

### ISSUE THREAD

**Issue by mipastgt** [gluonhq/substrate#336]
I don't know if this is a know limitation but with substrate it is not possible to play audio clips because you will get an unsatisfied link error caused by a missing glib-lite library.

macOS 10.14.6
GraalVM 19.3.1
client-maven-plugin 0.1.15
jfx 15-ea+1

**Comment by johanvos** [comment]
We don't have static builds for OpenJFX yet that include media for MacOS.
We're about to have support for Linux though, PR is at https://github.com/openjdk/jfx/pull/109

Feel free to open an issue on substrate if you want media support for MacOS too (which I assume you want ;) )

**Comment by mipastgt** [comment]
@johanvos Äh, isn't this the issue tracker for substrate?

**Comment by johanvos** [comment]
yes it is, I meant opening a specific issue, but it's easier if I just add it to the title of this -- sorry about the confusion!
